src/index.html

```diff
@@ -2,12 +2,503 @@
+            "
+            >
+              The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+            </div>
+
+            <div
+              class="
+                main__section__couple-img
+                main__section__couple-img--fifth
+                grid__item--1-2
+                grid__item--desktop--1-4
+              "
+            >
+            </div>
+
+            <div
+              class="
+                main__section__couple-img
+                main__section__couple-img--sixth
+                grid__item--3-6
+                grid__item--desktop--5-12
+              "
+            >
+            </div>
+
+            <h3
+              class="
+                main__section__title--secondary
+                grid__item--1-4
+                grid__item--desktop--1-6
+              "
+            >
+              Hydraulic disc brakes & Lightweight
+            </h3>
+
+            <div
+              class="
+                main__section__secondary-text
+                main__section__secondary-text--details
+                main__section__secondary-text--details--last
+                grid__item--1-6
+                grid__item--desktop--1-8
+              "
+            >
+              Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+            </div>
+
+            <button
+              class="
+                main__section__button
+                main__section__button--explore
+                grid__item--2-5
+                grid__item--desktop--5-8
+              "
+            >
+              Explore
+            </button>
+          </div>
+        </section>
+
+        <section class="main__section">
+          <div class="grid grid--desktop">
+            <h2
+              class="
+                main__section__title
+                grid__item--2-5
+                grid__item--desktop--4-9
+
+              "
+              id="contact"
+            >
+              Contact us
+            </h2>
+
+            <div
+              class="
+                grid__item--1-3
+                grid__item--desktop--1-5
+              "
+            >
+              <input
+                class="
+                  main__section__input
+
+                "
+                placeholder="Name"
+              >
+              <input
+                class="
+                  main__section__input
+                  grid__item--1-3
+                "
+                placeholder="Email"
+              >
+
+              <textarea
+                class="
+                  main__section__textarea
+                  grid__item--1-3
+                "
+                placeholder="Message"
+              ></textarea>
```

Comment:
Checklist 13th and 14th items

Reply:
Also styles look quite different to me
![image](https://user-images.githubusercontent.com/37161315/170645760-7234e7cf-08c2-42cc-93ed-7c5eeb58512e.png)